netbox/utilities/querydict.py

```diff
@@ -55,7 +55,7 @@ def prepare_cloned_fields(instance):
     for key, value in attrs.items():
         if type(value) in (list, tuple):
             params.extend([(key, v) for v in value])
-        elif value is not False and value is not None:
+        elif value is not None and (key.startswith('cf_') or value is not False):
```

Comment:
I'm afraid this isn't a tenable change: We cannot make arbitrary exceptions for certain types of fields. Instead, the form initialization logic may need to be updated to acknowledge explicit False (as opposed to undefined) values.

Reply:
The problem with updating the form initialisation logic is that we can't do that indiscriminately either. There may well be a `CharField` that receives the string `"False"`, in which case it isn't a very useful idea to modify it to the boolean value `False`.

So even when we modify the initialisation logic we still need to discriminate between field types that receive the data. Which is what I did now, this time on field type level.